doc/source/language/variables.rst

```diff
@@ -234,8 +235,9 @@ expression, the expression will not get executed if the calling
 function had an argument present in that position.  The expression
 only gets executed if the system needed to pad a missing argument.
 
-.. versionadded:: 0.18.3
-   Optional Parameters were added as a new feature in kOS 0.18.3
+.. note::
+    .. versionadded:: 0.18.3
+        Optional Parameters were added as a new feature in kOS 0.18.3
```

Comment:
It's not too hard to find all the `.. note::`s and change them all back if `versionadded` gets tweaked later. Can I suggest leaving them in `note`s so that they look right for now? Same idea as the bulleted lists as a workaround.